src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -161,9 +161,7 @@ const FileCard: React.FC<FileCardProps> = ({ file, removeFile, updateFile, toggl
                         return selectedLicense ? (
                           <StyledVerticalAlign>
                             <img src={selectedLicense.image} alt={selectedLicense.name} />
-                            <StyledLicenseName display="inline" variant="body1">
-                              {option}
-                            </StyledLicenseName>
+                            <StyledLicenseName display="inline">{option}</StyledLicenseName>
```

Comment:
skulle `StyledLicenseName` vært en extension av `NormalText` i stedet for `Typography`? (siden body1 er fjernet her)

Reply:
jepp. jeg kom borti problemet med display-prop her, og skulle tilbakestille endringene. Men egentlig bør vel NormalText også arve display=inline